sources/packages/web/src/views/student/application-offering-change-request/RequestedApplicationOfferingDetails.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header :title="changeRequest.studentFullName">
+        <template #subtitle
+          ><detail-header :headerMap="headerDetailsData"
+        /></template>
+        <template #status-chip>
+          <status-chip-application-offering-change
+            v-if="changeRequest.status"
+            :status="changeRequest.status"
+          />
+        </template>
+      </body-header>
+    </template>
+    <hr class="horizontal-divider" />
+    <h2 class="category-header-large primary-color">Application Details</h2>
+    <p v-if="changeRequestNotApproved">
+      Below displays the requested changes from your institution. You can
```

Comment:
as per figma `active application details` is in bold
![image](https://github.com/bcgov/SIMS/assets/77353155/1de8d393-585c-45f3-9c10-a3fae18982ba)

Reply:
we don't use `<b` we do have a class as per the Figma standard, I belive this is the one, pls double and use the class
![image](https://github.com/bcgov/SIMS/assets/77353155/d0ea71cd-1cf3-4121-ae0a-b29e58ebbbca)